### PR TITLE
Various tvg-id updates

### PR DESCRIPTION
**Pull request by AntiPontifex**

### CODE REVIEW

**Comment by AntiPontifex** [comment]
I used Google Translate, fwiw.

**Review by BellezaEmporium — COMMENTED** [review]
Looks apparently good though I can't confirm for the names as I don't speak Chinese.

**Comment by AntiPontifex** [comment]
As I indicated in the comments, the best I could do was Mandarin to English via Google. I think, unless someone else can prove otherwise,  it is good enough for approval.

**Comment by AntiPontifex** [comment]
This is a strange fail: I think someone should look into this.

**Comment by LaneSh4d0w** [comment]
Fixed the blank lines being removed as it's normal for all the playlists to have it
Fixed checking issues over cn and uk.

File: streams/ma.m3u [hunk 1]
```diff
@@ -27,7 +27,7 @@ http://streaming1.medi1tv.com/live/smil:medi1tv.smil/playlist.m3u8
 https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/tamazight_tv8_snrt/hls_snrt/tamazight_tv8_snrt.m3u8
 #EXTINF:-1 tvg-id="TeleMaroc.ma" status="online",Télé Maroc (1080p)
 https://api.new.livestream.com/accounts/27130247/events/8196478/live.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",Al Aoula International (360p)
+#EXTINF:-1 tvg-id="AlAoula.ma" status="blocked",Al Aoula International (360p)
```
Comment (LaneSh4d0w) [inline streams/ma.m3u:30]:
```suggestion
#EXTINF:-1 tvg-id="AlAoulaEurope.ma" status="blocked",Al Aoula International (360p)
```
Consider using the European prefix for the International stream as they have their local channel too (the same works for 2M). It is the only channel from the SNRT to have this subtility as it's the first channel.

**Review by BellezaEmporium — COMMENTED** [review]
See proposition.

**Review by BellezaEmporium — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Pull request merged**